### PR TITLE
Ita translation

### PR DESCRIPTION
**Pull request by Giuzy**
Translation and adjustments

### CODE REVIEW

File: Assets/XML/GameText/Units_CIV4GameText.xml [hunk 1]
```diff
@@ -4379,6 +4379,7 @@
 			<Plural>0:0:0:0:1:1:1:1</Plural>
 		</German>
 		<Russian>Ãðóïïà ëþäåé</Russian>
+    <Italian>Comitiva ancestrale</Italian>
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_UNIT_BANDEIRANTE_PEDIA</Tag>
```
Comment (Toffer90) [inline Assets/XML/GameText/Units_CIV4GameText.xml:4385]: The only thing I have to nitpick on here. tabulation ^^

Comment (Giuzy) [reply Assets/XML/GameText/Units_CIV4GameText.xml:4385]: For what I can see, there are also 4 blanks in the file Promotions_CIV4GameText.xml (sorry).

Comment (Giuzy) [reply Assets/XML/GameText/Units_CIV4GameText.xml:4385]: image

Comment (Giuzy) [reply Assets/XML/GameText/Units_CIV4GameText.xml:4385]: image

Comment (Giuzy) [reply Assets/XML/GameText/Units_CIV4GameText.xml:4385]: Nothing else I hope, checked multiple times

Comment (Toffer90) [reply Assets/XML/GameText/Units_CIV4GameText.xml:4385]: > Nothing else I hope, checked multiple times

Nothing else. ^^

Comment (Giuzy) [reply Assets/XML/GameText/Units_CIV4GameText.xml:4385]: well then this time it has gone better, but there' s still room to improve... :)

Comment (Toffer90) [reply Assets/XML/GameText/Units_CIV4GameText.xml]: Hmm, the tabulation is still wrong.... what text editor are you using?

Comment (Giuzy) [reply Assets/XML/GameText/Units_CIV4GameText.xml]: Ok it should be good now and from now on I know I can correct them myself. Thanks and good night!

**Pull request merged**